packages/components/molecules/f-restaurant-card/src/components/RestaurantCard.v1.vue

```diff
@@ -1,30 +1,34 @@
 <template>
-    <!-- NOTE: This is all placeholder markup, attributes and comments.
-    This is not indicative of the actual HTML tags and attributes we
-    will use (which will be much more accessible and semantic) -->
-    <section
-        :class="$style['c-restaurantCard']"
-        data-test-id="restaurantCard-component">
-        <a
-            :href="url"
-            @click="$emit('restaurant-card-clicked')">
-
-            <!-- background image -->
-            <img
-                src=""
-                alt=""
-                loading="lazy"
-                data-test-id="restaurant-cuisine-image">
-
+    <a
+        :href="url"
+        :class="[
+            $style['c-restaurantCard'],
+            $style[displayModeModifier],
+            { [$style['c-restaurantCard--with-img']]: !!imgUrl }]"
```

Comment:
Small naming thing – we tend to try and use `has` or `is` for stateful naming like this where it makes sense to (just a consistency thing)

```suggestion
            { [$style['c-restaurantCard--hasImg']]: !!imgUrl }]"
```